app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,50 @@
  *= require_tree .
  *= require_self
  */
+
```

Comment:
It's better not to add css to application.css - its intended purpose is to be a manifest file. It won't do any harm but some people consider it bad style.